cmd/vulcan-vulners/main.go

```diff
@@ -105,8 +105,17 @@ func severity(score float32) string {
 	}
 }
 
-// vulnerabilities retrieves the vulnerabilities affecting a software component from the vulners.com API.
-func vulnerabilities(p, s, v, t string) (*report.Vulnerability, error) {
+type vulnerFinding struct {
+	Score    float32
+	Resource report.ResourcesGroup
+}
+
+// buildVulnerFinding builds a vulner Finding querying the vulners.com API. The
+// resources of the finding contain the CVE'S found for the software component.
+// The Score of the finding contains the highest score found in the all the
+// CVE's.
+func buildVulnerFinding(p, s, v, t string) (*vulnerFinding, error) {
```

Comment:
it seems that the first parameter `s` is not being used, is this intended?

Reply:
It's used in the line 58, right?

Reply:
(from 121)

Reply:
Yes it is used there where Julian says.

Reply:
Sorry guys! I said the "first parameter" but then I named the second one... the parameter which is not being in use is the `p` (first argument), can you double check this, please?

Reply:
OK, now I see, you are right. That was already there and I need to investigate why I put that :D

Reply:
OK, that parameter contains the product name and looks like finally we are not using it for anything because in the places we are using the product name we are taking it from the nmap info.

I think we can just remove it.